be1-go/hub/standard_hub/mod.go

```diff
@@ -151,68 +181,71 @@ func (h *Hub) Receiver() chan<- socket.IncomingMessage {
 	return h.messageChan
 }
 
-// OnSocketClose implements hub.Hub
-func (h *Hub) OnSocketClose() chan<- string {
-	return h.closedSockets
+// AddServerSocket adds a socket to the sockets known by the hub and query a
+// catchup from the new server
+func (h *Hub) AddServerSocket(socket socket.Socket) error {
```

Comment:
I would rename it to something like `NotifyNewServer`. There we can think that it only takes note of a new server, where it in fact also catchup from it.